crates/packager/src/package/app/mod.rs

```diff
@@ -158,6 +158,7 @@ pub(crate) fn package(ctx: &Context) -> crate::Result<Vec<PathBuf>> {
                 m.notarization_credentials
                     .clone()
                     .ok_or_else(|| crate::Error::MissingNotarizeAuthVars)
+                    .or_else(|_| codesign::notarize_auth())
```

Comment:
instead of duplicating the call here we can just make the option fall through by using `.and_then` instead of `.map`,
```rs
 match config
            .macos()
            .and_then(|m| m.notarization_credentials.clone())
            .unwrap_or_else(codesign::notarize_auth)
```

Reply:
good call!

Reply:
Changed it to this:
```rust
        match config
            .macos()
            .and_then(|m| m.notarization_credentials.clone())
            .ok_or(crate::Error::MissingNotarizeAuthVars)
            .or_else(|_| codesign::notarize_auth())
        {
```